app/src/crypto.c

```diff
@@ -20,16 +20,18 @@
 
 uint32_t hdPath[HDPATH_LEN_DEFAULT];
```

Comment:
I was also wondering if this is the right place to define global arrays, and if there isn't another place in the code just should centralize and track all global variables (at least the arrays, say larger than 8 bytes)